app/test/account/like_test.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:test/test.dart';
+
+import '../shared/test_models.dart';
+import '../shared/test_services.dart';
+
+void main() {
+  group('/api/account/likes', () {
+    testWithServices('Like package', () async {
+      final client = createPubApiClient(authToken: hansUser.userId);
+      final rs = await client.getLikePackage('hydrogen');
+      expect(rs.package, 'hydrogen');
+      expect(rs.liked, false);
+
+      final rs2 = await client.likePackage('hydrogen');
+      expect(rs2.package, 'hydrogen');
+      expect(rs2.liked, true);
+
+      final rs3 = await client.getLikePackage('hydrogen');
+      expect(rs3.package, 'hydrogen');
+      expect(rs3.liked, true);
+    });
+
+    testWithServices('List package likes', () async {
+      final client = createPubApiClient(authToken: hansUser.userId);
+      final rs = await client.getLikes();
+      expect(rs.likedPackages.isEmpty, true);
+
+      final rs2 = await client.likePackage('hydrogen');
+      expect(rs2.package, 'hydrogen');
+      expect(rs2.liked, true);
+
+      final rs3 = await client.likePackage('helium');
+      expect(rs3.package, 'helium');
+      expect(rs3.liked, true);
+
+      final rs4 = await client.getLikes();
+      expect(rs4.likedPackages.length, 2);
+      expect(rs4.likedPackages[0].package, 'hydrogen');
+      expect(rs4.likedPackages[0].liked, true);
+      expect(rs4.likedPackages[1].package, 'helium');
+      expect(rs4.likedPackages[1].liked, true);
```

Comment:
Maybe we should be careful not to rely on the ordering here, that makes the test fragile. Unless you wish to document the ordering in the API docs, in which case that might be cool.

Maybe ordering by `created` is a great idea.